packages/dynamic-links/lib/modular/index.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { firebase } from '..';
+
+export const ShortLinkType = {
+  SHORT: 'SHORT',
+  UNGUESSABLE: 'UNGUESSABLE',
+  DEFAULT: 'DEFAULT',
+};
+
+/**
+ * @returns {import('.').DynamicLinks}
+ */
+export function getDynamicLinks() {
+  return firebase.dynamicLinks();
+}
+
+/**
+ *
+ * @param {import('.').DynamicLinks} dynamicLinks
+ * @param {import('..').FirebaseDynamicLinksTypes.DynamicLinkParameters} dynamicLinkParams
```

Comment:
Same mention I've made on other PRs about the typings in the params/returns - I don't think they should be programmatic? Would apply to all APIs in this file

```suggestion
 * @param {DynamicLinks} dynamicLinks
 * @param {FirebaseDynamicLinksTypes.DynamicLinkParameters} dynamicLinkParams
```